snapshot-labs/sx-ui
# feat: Add mixpanel tracking

### Summary



Not sure what the best approach is for Vue 3. There is very little downloads for `vue-mixpanel` and it looks very Vue 2. I figured I could create a composable instead.

Closes: #768

### How to test

1. Go to space create page
2. Check on Mixpanel dashboard if tracking is working

<img width="1763" alt="image" src="https://github.com/snapshot-labs/sx-ui/assets/1968722/09064e64-b451-4f6c-bc90-69324421023f">
